cmd/serverless-init/cloudservice/appservice.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cloudservice
+
+import (
+	"os"
+)
+
+// AppService has helper functions for getting specific Azure Container App data
+type AppService struct{}
+
+const (
+	WebsiteName = "WEBSITE_SITE_NAME"
+	RegionName  = "REGION_NAME"
+	RunZip      = "APPSVC_RUN_ZIP"
+)
+
+// GetTags returns a map of Azure-related tags
+func (a *AppService) GetTags() map[string]string {
+	appName := os.Getenv(WebsiteName)
+	region := os.Getenv(RegionName)
+
+	return map[string]string{
```

Comment:
I didn't include `revision` here like the ContainerApp, not sure if that's right but just calling it out